bumble/gatt.py

```diff
@@ -346,8 +346,11 @@ def write_encoded_value(self, connection, value):
     async def read_decoded_value(self):
```

Comment:
For `write_encoded_value()` above, are the same changes not necessary?

Reply:
The other `write_encoded_value` in this adapter class is for the GATT server side (when you want to get a callback for a received WRITE), which doesn't have a `with_response` parameter, since that's not passed to the callback handler (the stack will send a response or not, depending on whether the operation is a WRITE request or command, but the handler doesn't know that).

Reply:
(the `CharacteristicAdapter` class is a general purpose class that can be used both on the client and server side)